torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -808,27 +839,25 @@ def create_wrapped_node(self, tx, args, kwargs):
         )
 
         body_node = make_attr(tx, body_name)
-        p_args = (
-            body_node,
-            *(arg.as_proxy() for arg in args[1:]),
-            *(arg for arg in body_lifted_freevars.keys()),
-        )
+
+        # Since, we call `speculate_subgraph` with `manually_set_subgraph_inputs=False`,
+        # all the arguments are lifted.
+        lifted_args = tuple(arg for arg in body_lifted_freevars.keys())
+
+        proxy_args = (body_node,) + lifted_args
         example_value = pytree.tree_map_only(
             torch.fx.Proxy,
             lambda a: a.node.meta["example_value"],
             body_r.as_proxy(),
         )
-        _, p_kwargs = proxy_args_kwargs([], kwargs)
-        return p_args, p_kwargs, example_value
+
+        return proxy_args, {}, example_value
 
     def call_function(
         self, tx, args: "List[VariableTracker]", kwargs: "Dict[str, VariableTracker]"
     ) -> "VariableTracker":
-        from . import ConstantVariable
         from .builder import wrap_fx_proxy
 
-        self.check_kwargs(kwargs, ConstantVariable)
```

Comment:
Removing the check as `speculate_subgraph` does it. 

Also, `check_kwargs` does assert which actual breaks dynamo instead of `graph-breaking`.
https://github.com/pytorch/pytorch/blob/fbe2a7e50a940ba7a12b003241a2699f7a731afb/torch/_dynamo/variables/higher_order_ops.py#L223-L227

Reply:
Uh oh, that's a good catch

Reply:
Done : https://github.com/pytorch/pytorch/issues/104884